server/sonar-server/src/main/java/org/sonar/server/settings/ws/ValuesAction.java

```diff
@@ -0,0 +1,291 @@
+/*
+ * SonarQube
```

Comment:
I think the name `setValues` field is not clear enough. It's hard to know what it is. Why not use the existing values field? I wonder what is the best: having one field values (for value, values and setValues) or as you did with 3 fields